Tests/PointFreeTests/AuthTests.swift

```diff
@@ -16,7 +16,7 @@ class AuthTests: TestCase {
     let conn = connection(from: request)
     let result = conn |> siteMiddleware
 
-    assertSnapshot(matching: result)
+    assertSnapshot(matching: result.perform())
```

Comment:
once we do the snapshotting function (as opposed to protocol) we could prob get rid of these...